apps/sms/views/shared/js/activity_handler.js

```diff
@@ -353,7 +353,8 @@ var ActivityHandler = {
 
```

Comment:
you need to remove `Threads` from the globals line, see https://treeherder.mozilla.org/logviewer.html#?job_id=1912132&repo=gaia